### PR TITLE
Yarn & Node.js upgrades

### PR DESCRIPTION
**Pull request by mutantcornholio**

### CODE REVIEW

File: .yarn/install-state.gz
Comment (rzadp) [inline .yarn/install-state.gz:1]: Should not be commited, right?

Comment (mutantcornholio) [reply .yarn/install-state.gz:1]: 🤦 Good catch, thanks!

Comment (mutantcornholio) [reply .yarn/install-state.gz:1]: I've probably staged it first, and only then added to .gitignore...

File: .yarn/releases/yarn-4.3.0.cjs
Comment (rzadp) [inline .yarn/releases/yarn-4.3.0.cjs:1]: And this - should it be committed?

Comment (mutantcornholio) [reply .yarn/releases/yarn-4.3.0.cjs:1]: Yes. Vendoring yarn together with code, will ensure correct version, doesn't matter what's in environment (most likely, there will be `1.22.22` in environment).

**Review by rzadp — APPROVED** [review]

**Pull request merged**